### PR TITLE
Added Connected Sites props, stories & tests to AccountListItem

### PR DESCRIPTION
**Pull request by vthomas13**
## Explanation
The disconnect icon is a placeholder for now
Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1226

### After
image

## Manual Testing Steps

These changes are unimplemented, so most testing will be through storybook
## Pre-merge author checklist

- [X] I've clearly explained:
  - [X] What problem this PR is solving
  - [X] How this problem was solved
  - [X] How reviewers can test my changes
- [X] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/components/multichain/account-list-item/account-list-item.js [hunk 1]
```diff
@@ -217,18 +221,21 @@ export const AccountListItem = ({
...
-            <UserPreferencedCurrencyDisplay
-              ethNumberOfDecimals={MAXIMUM_CURRENCY_DECIMALS}
-              value={identity.balance}
-              type={SECONDARY}
-            />
-          </Text>
+          {showSecondaryBalance ? (
+            <Text
+              variant={TextVariant.bodySm}
+              color={Color.textAlternative}
```
Comment (DDDDDanica) [inline ui/components/multichain/account-list-item/account-list-item.js:227]:
```suggestion
              color={TextColor.textAlternative}
```
I know this was there originally but if you can, would you mind making this change?

Comment (vthomas13) [reply ui/components/multichain/account-list-item/account-list-item.js:227]: Good catch!

**Review by darkwing — DISMISSED** [review]
This is excellent!  One tiny ask:  can you add a storybook item for Hardware Wallet + "Active"?

**Comment by vthomas13** [comment]
> This is excellent! One tiny ask: can you add a storybook item for Hardware Wallet + "Active"?

Added in [95ecf4d](https://github.com/MetaMask/metamask-extension/pull/20779/commits/95ecf4db8a24da799810b5cf3f7dde32f5f8eabf)

File: ui/components/multichain/account-list-item/account-list-item.js [hunk 2]
```diff
@@ -239,35 +246,56 @@ export const AccountListItem = ({
...
-      <AccountListItemMenu
-        anchorElement={accountListItemMenuElement}
-        identity={identity}
-        onClose={() => setAccountOptionsMenuOpen(false)}
-        isOpen={accountOptionsMenuOpen}
-        isRemovable={keyring?.type !== KeyringType.hdKeyTree}
-        closeMenu={closeMenu}
-      />
+      {showDisconnectIcon ? (
+        <>
```
Comment (darkwing) [inline ui/components/multichain/account-list-item/account-list-item.js:260]: Are these `<>` necessary?

File: ui/components/multichain/account-list-item/account-list-item.js [hunk 3]
```diff
@@ -239,35 +246,56 @@ export const AccountListItem = ({
...
+          <ButtonIcon
+            iconName={IconName.Slash}
+            size={IconSize.Sm}
+            data-testid="account-list-item-disconnect-buttonicon"
```
Comment (darkwing) [inline ui/components/multichain/account-list-item/account-list-item.js:264]: At present this doesn't do anything -- should there be an onClick here?

Comment (vthomas13) [reply ui/components/multichain/account-list-item/account-list-item.js:264]: We are still tentative with the design on this, but I think the disconnect Icon will be removed completely, according to @kylefiedler  
[Figma](https://www.figma.com/file/LDlI7yjKBNIY60MtQv9QS0/Amon-Hen%2FV1?type=design&node-id=4878%3A841425&mode=design&t=uMbufVJ2quRetis8-1)

**Pull request closed without merging**